config/core/test-configs.yaml

```diff
@@ -2025,6 +2025,13 @@ test_configs:
   - device_type: imx6q-sabrelite
     test_plans:
       - baseline
+      - kselftest-filesystems
+      - kselftest-futex
+      - kselftest-lib
+      - kselftest-livepatch
+      - kselftest-lkdtm
+      - kselftest-rtc
```

Comment:
The test job was run but all rtc test cases failed:
https://staging.kernelci.org/test/plan/id/623378e36d56c7a3c34b3130/

Reply:
Remove rtc from here. It was mistakenly added. This PR is not related to platform tests.